packages/web/components/swap-tool/order-type-selector.tsx

```diff
@@ -0,0 +1,113 @@
+import { Menu, Transition } from "@headlessui/react";
+import classNames from "classnames";
+import { parseAsStringLiteral, useQueryState } from "nuqs";
+import React, { Fragment, useMemo } from "react";
+
+import { Icon } from "~/components/assets";
+import { SpriteIconId } from "~/config";
+import { useTranslation } from "~/hooks";
+
+interface UITradeType {
+  id: "market" | "limit" | "recurring";
```

Comment:
I think we can leave out "recurring" for now.

Reply:
[7292db4](https://github.com/osmosis-labs/osmosis-frontend/pull/3320/commits/7292db403df0dde7aa6409b806c06ca7bce4c92c)